data-raw/organizations/DIGO/prepare-DIGO.R

```diff
@@ -0,0 +1,56 @@
+# DIGO Preparation Script
+
+# The Death of International Organizations dataset by Eilstrup-Sangiovanni
+# contains information on IGOs created since 1815 
+# and the different ways some of these have ceased to exist.
+
+# This is a template for importing, cleaning, and exporting data
+# ready for many packages universe.
+
+# Stage one: Collecting data
+DIGO <- readr::read_csv("data-raw/organizations/DIGO/io_dataDID-1.0-RIO.csv")
+
+# Stage two: Correcting data
+# In this stage you will want to correct the variable names and
+# formats of the 'DIGO' object until the object created
+# below (in stage three) passes all the tests.
+# We recommend that you avoid using one letter variable names to keep
+# away from issues with ambiguous names down the road.
+DIGO <- as_tibble(DIGO) %>%
+  manydata::transmutate(igoID = ioname,
```

Comment:
Same variable name as in the cow organisations dataset?

Reply:
I wonder if we should we create an ID function to code IGO names to avoid (the best we can) mismatches when consolidating data?

Reply:
We could create something like a manyID for organisations/non-state actors, which would also be useful to distinguish between abbreviated state names and IGO/other actors' names especially when we include more data on other actors. At the moment I've used `igoID` for all the abbreviated names of IGOs. There is some overlap among the datasets because most of them used the abbreviations in the COW dataset where possible.